index.html

```diff
@@ -239,6 +239,51 @@ <h3 class="sk">
         </li>
       </ul>
     </section>
+    <section class="contact">
+      <div class="announcement">
+        <p>I'm alway interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      </div>
+      <form action="#" method="POST" class="form">
+        <ul>
+          <li>
+            <input type="text" placeholder="Full name">
+          </li>
+          <li>
+            <input type="email" placeholder="Email address">
```

Comment:
- [ ] Kindly please add the `required` attribute in your *input* email field because without it any user can send a mistake sending an email to you which actually will result in a waste of resources. Please look at the below example.

> Suppose you have a website which sells some products and people have some queries before buying any product to get the leads and get an instant email from the customer you will need better email services which cost $10 per 1000 emails. Now a genuine user asks the relevant things and maybe your customer gets converted and ready to buy your product then in that case your expense on email services will consider valuable but in the worst-case scenario, a user who just plays with your website and miss-use your resources will decrease your sales and also put unnecessary bill on you.  


*I hope the above example helps you why we need to require the `required` attribute in the form.*

### Requirement

![image](https://github.com/wacoo/Portfolio-Project/assets/66476329/5bee4c2f-4e48-440c-bfb4-c0b9bc269857)